app/helpers/fhir_helper.rb

```diff
@@ -0,0 +1,329 @@
+# frozen_string_literal: true
+
+# Helper module for FHIR translations
+module FhirHelper # rubocop:todo Metrics/ModuleLength
+  # Returns a representative FHIR::Patient for an instance of a Sara Alert Patient. Uses US Core
+  # extensions for sex, race, and ethnicity.
+  # https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-patient.html
+  def patient_as_fhir(patient)
```

Comment:
Maybe add initial check to verify that `patient` is not nil.

Reply:
Added https://github.com/SaraAlert/SaraAlert/pull/521/commits/3258b7a3f622672dcbab96490cb7fc30c257c27f